setup.py

```diff
@@ -7,14 +7,14 @@
 
 setup(
     name='edx-oauth-client',
-    version='2.0.2',
+    version='2.1.0',
     description='Client OAuth2 from edX installations',
     author='edX',
     url='https://github.com/raccoongang/edx_oauth_client',
     install_requires=[
-        "Django>=2.2,<2.3",
-        "social-auth-core>=1.7,<2.0",
-        "social-auth-app-django>=2.1,<3.0",
+        "Django",
+        "social-auth-core",
+        "social-auth-app-django",
```

Comment:
Note: `requirements.txt` reflects versions used by Maple (tested and compatible). We don't want to reinstall existing `edx-platform` packages during the deployment, so I don't think we are gaining anything by having these versions pinned in `setup.py`.